### PR TITLE
Fix fatal error due to missing function

### PR DESCRIPTION
**Pull request by cristian-ungureanu**
Tried to make a fix for https://github.com/anhskohbo/wp-cli-themecheck/issues/6, I don't know if it's the best solution, any feedback would be much appreciated.

### CODE REVIEW

**Comment by selul** [comment]
@anhskohbo would love to know your thoughts on this, there are many people using this package, and your help in merging this will be very much appreciated 💯

File: command.php [hunk 1]
```diff
@@ -139,8 +139,33 @@ private function themecheck( $theme, $path ) {
 			}
 
 			// Run Themecheck.
-			$data = tc_get_theme_data( $path . '/style.css' );
-			$success = run_themechecks( $datafiles['php'], $datafiles['css'], $datafiles['other'] );
+			if ( ! class_exists( 'WP_Theme' ) ) {
```
Comment (anhskohbo) [inline command.php]: @cristian-ungureanu actually there's a variable (`$theme`) instance of WP_Theme from __invoke method above.
You just need to pass it to `themecheck` method.

**Comment by anhskohbo** [comment]
@selul thank you for your concern. a new version will release soon.

File: command.php (hunk 1 above)
Comment (cristian-ungureanu) [reply command.php]: Fixed it, thanks for having a look over this!

**Pull request merged**